### PR TITLE
feat(commands) - extend from Global (User) scope

### PR DESCRIPTION
**Pull request by ttodua**
when you have workspace-scope settings, it overrides global(user) scope settings:
image

This is very necessary feature so we can add `true` in workspace-scope settings to inherit global-scope commands too, so we would have both of commands array in status-bar

### CODE REVIEW

**Comment by ttodua** [comment]
@seunlanlege can you please take a look? it's easy PR

**Comment by ttodua** [comment]
@seunlanlege can you just comment your thoughts, whether you'll review this at all?
if not, please let me know what is the license of this extension, so i could fork it and push a separate extension

File: package.json [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-	"version": "1.2.2",
+	"version": "1.2.3",
 	"publisher": "seunlanlege",
 	"repository": {
 		"type": "git",
 		"url": "https://github.com/SeunLanLege/vscode-action-buttons.git"
 	},
 	"engines": {
-		"vscode": "^1.27.2"
+		"vscode": "^1.75.0"
```
Comment (seunlanlege) [inline package.json:12]: why bump this?

Comment (ttodua) [reply package.json:12]: maybe not needed, I was following this info: https://stackoverflow.com/a/75303487/2377343 trying alter activation events, but you can remove that maybe and return to old version.

Comment (seunlanlege) [reply package.json:12]: suggest we try to maintain the maximum compatibility for the extension, pls revert this

Comment (ttodua) [reply package.json]: > suggest we try to maintain the maximum compatibility for the extension, pls revert this

done

**Review by seunlanlege — APPROVED** [review]

**Comment by ttodua** [comment]
@seunlanlege When do you plan to publish new version of extension for VSCODE ?

**Comment by seunlanlege** [comment]
Sorry about this, somehow skipped my mind

**Comment by seunlanlege** [comment]
Actually i just realized i tried to publish a new version a while ago, but i couldn't figure out how to create an access token on the azure dashboard (not even sure what azure has to do with this). Anyways, Microsoft bad, use zed. Cheers

**Comment by ttodua** [comment]
@seunlanlege what if you build the package (vsix) locally and just upload in dashboard ( https://marketplace.visualstudio.com/manage/publishers/ ) ?

image

**Comment by seunlanlege** [comment]
> @seunlanlege what if you build the package (vsix) locally and just upload in dashboard ( https://marketplace.visualstudio.com/manage/publishers/ ) ?
> 
> image

I've updated using this method

**Pull request merged**